forge-gui/res/cardsfolder/upcoming/demolition_field.txt

```diff
@@ -0,0 +1,9 @@
+Name:Demolition Field
+ManaCost:no cost
+Types:Land
+A:AB$ Mana | Cost$ T | Produced$ C | SpellDescription$ Add {C}.
+A:AB$ Destroy | Cost$ 2 T Sac<1/CARDNAME> | ValidTgts$ Land.nonBasic+OppCtrl | TgtPrompt$ Select target nonbasic land an opponent controls | SubAbility$ DBSearch | AILogic$ GhostQuarter | SpellDescription$ Destroy target nonbasic land an opponent controls. That land's controller may search their library for a basic land card, put it onto the battlefield, then shuffle. You may search your library for a basic land card, put it onto the battlefield, then shuffle.
+SVar:DBSearch:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | DefinedPlayer$ TargetedController,You | ChangeType$ Land.Basic | ChangeNum$ 1 | StackDescription$ That land's controller may search their library for a basic land card, put it onto the battlefield, then shuffle. You may search your library for a basic land card, put it onto the battlefield, then shuffle.
```

Comment:
I don't think DefinedPlayer supports that

Reply:
It allowed us both to seach for a basic land but I should probably test it with multiple opponents

Reply:
Tested it with 3 players and everyone got to search like Field of Ruin. Any ideas how to fix ?